rest_framework_jwt/serializers.py

```diff
@@ -169,3 +170,9 @@ def validate(self, attrs):
             'token': jwt_encode_handler(new_payload),
             'user': user
         }
+
+
+class DeviceSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Device
+        fields = ['permanent_token', 'jwt_secret', 'created', 'name', 'details', 'last_request_datetime']
```

Comment:
WHat?! you return JWT secret to the world? 
http://getblimp.github.io/django-rest-framework-jwt/#jwt_secret_key Please check documentation what JWT secret is.